src/js/util.js

```diff
@@ -290,6 +301,8 @@ exports.isFinite = isFinite;
 exports.isFunction = isFunction;
 exports.isBuffer = isBuffer;
 exports.isRegExp = isRegExp;
+exports.isDate = isDate;
+exports.isError = isError;
 exports.isArray = Array.isArray;
```

Comment:
But why we export it here? Why Array.isArray directly?

Reply:
This seems to be exported before from IoT.js.